prerendering.bs

```diff
@@ -641,6 +642,12 @@ This stores which client hints each origin has opted into receiving, until it ca
   1. Otherwise, [=map/set=] [=Accept-CH cache=][|origin|] to |hintSet|.
 </div>
 
+<h3 id="interaction-with-clear-site-data-clear-cache">Interaction with Clear-Site-Data Clear Cache</h3>
+
+Modify [$clear cache for origin$] to add the following step at the end.
+1. [=list/For each=] [=navigable/top-level traversable=] |traversable| in the user agent's [=top-level traversable set=]: 
+  1. If |traversable|'s [=navigable/active document=]'s [=Document/origin=] is [=same origin=] with |origin|, and |traversable| is a [=prerendering traversable=], then [=destroy a top-level traversable|destroy=] |traversable|.
```

Comment:
Use `<var ignore>` here to bypass the build error. This is unfortunately necessary very often when writing these kind of patch specs.

Reply:
It was actually the one in the algorithm.

I pushed a commit which fixes the build errors. Note that you can run the build locally using `make prerendering.html`. But it's all good now, so I'll merge!